DataProviders/src/main/java/com/softserveinc/reviewer/resources/ElasticSearchResource.java

```diff
@@ -27,11 +23,10 @@ public ElasticSearchResource(ElasticSearchService elasticSearchService) {
     public Response getReviews(@QueryParam("type") String type, @QueryParam("client") String destinationClientId,
                                @QueryParam("subjectProduct.externalId") String productId) {
 
-        List<Review> reviews = elasticSearchService.getReviews(type, destinationClientId, productId);
-        if(reviews.isEmpty()) {
+        ElasticSearchResponse response = elasticSearchService.getReviews(type, destinationClientId, productId);
```

Comment:
That shouldn't be a big deal, but I personally prefer to receive ```List<Review>``` (and other simple models) from the services, rather then ```xxxResponse```. And just build response instance for the sake of returning response to the request.